templates/ai-ml.html

```diff
@@ -1,374 +1,417 @@
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/795f2fc2-knative-logo.png" alt="KNative"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/20a0d896-pytorch-logo.png" alt="PyTorch"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/86e6c732-seldon-logo.png" alt="Seldon"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/c6a757b1-pachyderm-logo.png" alt="Pachyderm"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/31c63bdf-onnx%20logo.png" alt="ONNX"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/ddbc4d23-kserve-logo.png" alt="KServe"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/73accfaf-nvidia-triton-server-logo.png" alt="Nvidia Triton Server"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/797287f7-grafana-logo.png" alt="Grafana"></div>
+{% block outer_content %}
+  <section class="p-suru--25-75 p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>オープンソースAIでモデルを実運用環境へ</h1>
+      </div>
+      <div class="col">
+        <div class="p-image-wrapper u-hide--small">
+          {{ image(url="https://assets.ubuntu.com/v1/e549fdc0-Canonical-AI-ML-Illustrations-v5-03.png",
+                    alt="",
+                    width="600",
+                    height="400",
+                    hi_def=True,
+                    attrs={"class": "p-image-container__image"},
+                    loading="lazy") | safe
+          }}
+        </div>
+        <p>
+          安全でサポート付きのCanonical MLOpsを使用すれば、企業グレードのAIプロジェクトを構築できます。UbuntuワークステーションでCharmed KubeflowまたはCharmed MLFlowを使用して開発し、スタックの随所に装備されたオープンソースツールでスピーディーに拡張します。
+        </p>
+        <hr class="is-muted" />
+        <a href="/contact-us" class="p-button--positive">お問い合わせ</a><a class="p-button" href="/engage/mlops-guide-jp">MLOpsガイドを読む</a>
       </div>
-      <p><a href="https://ubuntu.com/download/iot" class="p-button is-wide p-link--external">Ubuntu for IoTを入手する</a></p>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-strip--light is-bordered">
-  <div class="row">
-    <h2 style="max-width: none;">CanonicalのAIおよび機械学習ソリューションの特長...</h2>
-  </div>
-  <div class="row u-equal-height">
-    <div class="p-card col-6">
-      <div class="p-card__header">
-        <h3>プライベートクラウドとHPCのアーキテクチャ</h3>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
```

Comment:
This is the new recommendation from Vanilla, hasn't been pushed to prod yet (I don't think) but pr [here](https://github.com/canonical/vanilla-framework/pull/5241/files) (same for all section separators below)
```suggestion
      <hr class"u-no-margin--bottom" />
```